ludwig/utils/time_utils.py

```diff
@@ -68,15 +68,15 @@ def proc(self):
         return time.clock() - self._proc
 
     def tic(self):
-        """Like Matlab tic/toc for wall time and processor time"""
+        """Like Matlab tic/toc for wall time and dataframe time"""
```

Comment:
this and the next replace are likely wrong :)

Reply:
I blame PyCharm!